tests/puppeteer/campaigns/sanity/catalogFO/01_filterProducts.js

```diff
@@ -0,0 +1,49 @@
+// Using chai
+const {expect} = require('chai');
+
+// Importing pages
+const HomePage = require('../../../pages/FO/home');
+const ProductPage = require('../../../pages/FO/product');
+
+let page;
+let homePage;
+let productPage;
+let allProductsNumber = 0;
+
+// creating pages objects in a function
+const init = async () => {
+  page = await global.browser.newPage();
```

Comment:
Is it possible to access browser without using a global here?  (just asking, if it's possible it would be nice)

(same for other usages of global)

Reply:
This is a problem that we are discussing in the qa team, and should be resolved in another PR.
FYI : it's not linked to ths PR, there are already some tests with this method and we are trying to find a way to avoid this bad practice and have something that fill in puppeteer + mocha architecture